### PR TITLE
Martyrdoms

### PR DESCRIPTION
**Pull request by eu-genia**

### CODE REVIEW

File: 1001-2000/LIT1010Abuqer.xml [hunk 1]
```diff
@@ -39,6 +52,14 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          </xi:include>
       </encodingDesc>
       <profileDesc>
+         <abstract>
+            <p>Gadl translated from Arabic by the efforts of abbā Salāmā (1348-1388); see 
+               colophon in <ref type="mss" corresp="EMML1826"/>,f. 49vb, <quote xml:lang="gez">ወጽህቀ፡ ባቲ፡ ለአዕልዎታ፡ እምዓረቢ፡ ለግዘ፡ አበነ፡ ቅዱስ፡ አባ፡ ሰላማ፡ ጸሎቱ፡ ትዕቀበነ፡ አሜን።=።</quote>; colophon in <ref type="mss" corresp="BLorient687"/>
```
Comment (MarcinKrawczuk) [inline 1001-2000/LIT1010Abuqer.xml:57]: ለግዕዝ? አቡነ?

File: 6001-7000/LIT6349MartyrdomAkaweh.xml [hunk 2]
```diff
@@ -47,10 +56,18 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
         <revisionDesc>
             <change who="MKr" when="2021-01-19">Created entity</change>
+            <change when="2021-12-29" who="ES">updated with notes from Alessandro Bausi</change>
         </revisionDesc>
     </teiHeader>
     <text>
         <body>
+            <div type="edition" xml:lang="gez">
+                <div type="textpart" subtype="incipit">
+                    <ab>በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ አሐዱ፡ አምላክ።=። ዝውእቱ፡ ገድል፡ ወስምዕ፡ ዘቅዱስ፡ አባ፡ አካውሕ፡ ሰማዕቱ፡ ለእግዚእነ፡ ኢየሱስ፡ ክርስቶስ፡ ዘፈጸመ፡ ገድሎ፡ ንጹሐ፡ አመ፡ ፳ወ፰ለወርኃ፡ ጥር፡ ጸሎቱ፡ ወበረከቱ፡ ተሀሉ፡ ምስሌነ፡ አሜን። ወኮነ፡ በውእቱ፡ መዋዕል፡ ስደት፡ በዓመተ፡ ፳፪እመንግሥተ፡ ዲዮግልዲያኖስ፡ አማፂ። አመ፡ ፳ወ፮ለወርኃ፡ ትኅሣሥ፡ ሀሎ፡ ክልክልያኖስ፡ ዘመጽአ፡ እምኀበ፡ ንጉሥ፡ ወከመ፡ ወሀቦ፡ ስልጣነ፡ ከመ፡ ኵሉ፡ ሰብእ፡ እለ፡ ኢይሰግዱ፡ ለአማልክተ፡ ንጉሥ፡ ይቅትሎሙ፡ በዘዘ፡ ዚአሁ፡ ሞት፡ እኩየ።=።
```
Comment (MarcinKrawczuk) [inline 6001-7000/LIT6349MartyrdomAkaweh.xml:66]: ታኅሣሥ?

File: 6001-7000/LIT6349MartyrdomAkaweh.xml [hunk 3]
```diff
@@ -47,10 +56,18 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                    </ab>
+                    <note>quoted after <ref type="mss" target="BLorient692"/>, f. 116rb</note>
```
Comment (MarcinKrawczuk) [inline 6001-7000/LIT6349MartyrdomAkaweh.xml:68]: Delete space and comma?

File: new/LIT6528MartyrdomAbrahamGarga.xml [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change who="ES" when="2021-12-29">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition" xml:lang="gez">
+                <div type="textpart" subtype="incipit">
+                    <ab>በስመ፡ አብ፡... አመ፡ ዐሡሩ፡ ወሰሙኑ፡ ለግንቦት፡ ገድል፡ ዘምልዕት፡ ኩሎ፡ መንክራተ፡ ወፍድፍድናተ፡ ዘለአበዊነ፡ ዘበአማን፡
```
Comment (MarcinKrawczuk) [inline new/LIT6528MartyrdomAbrahamGarga.xml:58]: ኵሎ፡?

File: new/LIT6528MartyrdomAbrahamGarga.xml [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+        <revisionDesc>
+            <change who="ES" when="2021-12-29">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition" xml:lang="gez">
+                <div type="textpart" subtype="incipit">
+                    <ab>በስመ፡ አብ፡... አመ፡ ዐሡሩ፡ ወሰሙኑ፡ ለግንቦት፡ ገድል፡ ዘምልዕት፡ ኩሎ፡ መንክራተ፡ ወፍድፍድናተ፡ ዘለአበዊነ፡ ዘበአማን፡ 
+                        ብፁዓዊያን፡ አባ፡ አብርሃም፡ ወአባ፡ ገርጋ፡ መነኮሳን፡ ልቡሳነ፡ መንፈስ፡ ዘእምነ፡ ደብር፡ ቅዱስ፡ ደብረ፡ አቡነ፡ መቃርስ፡ ዘውዕቱ፡ መዳልወ፡
```
Comment (MarcinKrawczuk) [inline new/LIT6528MartyrdomAbrahamGarga.xml:59]: መነኮሳት?

File: new/LIT6529GadlaAbsadiAlaniqos.xml [hunk 6]
```diff
@@ -0,0 +1,65 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6529GadlaAbsadiAlaniqos" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Combat of ‘abbā Absādi and bishop ‘Alāniqos from Egypt</title>
```
Comment (MarcinKrawczuk) [inline new/LIT6529GadlaAbsadiAlaniqos.xml:8]: Maybe it's just the way it displays but I use ʾ for አ

**Review by MarcinKrawczuk — APPROVED** [review]

File: 1001-2000/LIT1010Abuqer.xml (hunk 1 above)
Comment (eu-genia) [reply 1001-2000/LIT1010Abuqer.xml:57]: I copypasted from Alessandro's file, so would leave it for now

File: new/LIT6528MartyrdomAbrahamGarga.xml (hunk 5 above)
Comment (eu-genia) [reply new/LIT6528MartyrdomAbrahamGarga.xml:59]: መነኮሳን in the ms

**Pull request merged**